gateway-messages/src/lib.rs

```diff
@@ -207,6 +222,27 @@ pub enum SensorDataMissing {
     DeviceTimeout,
 }
 
+/// Request to the CMPA
+#[derive(
+    Debug, Clone, Copy, PartialEq, Eq, SerializedSize, Serialize, Deserialize,
+)]
+pub enum RotRequest {
+    ReadCmpa,
+    ReadCfpa(CfpaPage),
+}
+
+/// Specific CFPA page
+#[derive(
+    Debug, Clone, Copy, PartialEq, Eq, SerializedSize, Serialize, Deserialize,
+)]
+pub enum CfpaPage {
+    /// Currently active page
+    Active,
+    /// Page that will become active on reset
+    Pending,
+    // We could add variants for ping/pong/scratch pages here if needed
+}
```

Comment:
I think we should be more explicit to cover all cases
```
enum CfpaPage {
     Active, // HIghest version between ping/pong
     Alternate, // Lower version between ping/pong
     Scratch, // Data to be applied on next reboot
}
```
Given the difficulty in getting data out of the RoT, I'd like to guarantee we can actually get all possible data we want and then leave interpretation of that elsewhere. My fear is that the RoT tries to calculate what will happen with "pending", something goes wrong and then we're stuck with confusing data when it doesn't get applied.

Reply:
Sure, I can do that.  Do you want `Active` / `Alternate`, or should we get even more manual and report `Ping` / `Pong` (leaving the host to interpret which one is active)?

Reply:
Okay, switched to `Active` / `Inactive` / `Scratch` in 39455d696e1367377bec6e8c102cf034a837ac1f